dlp/snippets/deid/deid_test.go

```diff
@@ -100,3 +100,34 @@ func TestDeidentifyDateShift(t *testing.T) {
 		})
 	}
 }
+
+func TestDeIdentifyWithWordList(t *testing.T) {
+	tc := testutil.SystemTest(t)
+	tests := []struct {
+		input    string
+		infoType string
+		wordList []string
+		want     string
+	}{
+		{
```

Comment:
suggestion: there is a lot of set up here for a single test case. I would either add more test cases or forego using the table-driven test design altogether.

Reply:
changed it.